<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>우편물 접수</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>title_big.png</normaloff>title_big.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="minimumSize">
           <size>
            <width>198</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color : rgba(0,0,0,0%)</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'맑은 고딕'; font-size:14pt; font-weight:696;&quot;&gt;묶음우편물 확인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_selectallrow">
          <property name="text">
           <string>전체선택</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>묶음우편물 목록</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="table_grouppostlist">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>접수날짜</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>사원번호</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>접수자명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>부서명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>보내는사람</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>받는사람</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>주소</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>제목</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>수량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>종류</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>긴급여부</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>가로길이</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>세로길이</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>우편중량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>높이</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>메모</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="action_closewindow"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_closewindow">
   <property name="text">
    <string>창 닫기</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>btn_selectallrow</tabstop>
  <tabstop>table_grouppostlist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
